### PR TITLE
Fixed BleBox wLightBox

### PR DESCRIPTION
**Pull request by d4m**

### CODE REVIEW

File: hardware/BleBox.h [hunk 1]
```diff
@@ -44,6 +44,10 @@ class BleBox : public CDomoticzHardwareBase
 	boost::shared_ptr<boost::thread> m_thread;
 	std::map<const std::string, const int> m_devices;
 	boost::mutex m_mutex;
+	
+	float LimitlessRGBWcHueState;
```
Comment (gizmocuz) [inline hardware/BleBox.h]: Could you describe what you 'fixed' ?
Member functions should start with m_

Comment (d4m) [reply hardware/BleBox.h]: "Limitless" support did not work at all(only "LedOn" worked), I rewritten and tested it on my wLightBox device.

I did not create new functions, it's just variables to remember the state in the session.

**Comment by gizmocuz** [comment]
Sorry, i meant please start Member variables with a m_ (m underscore)

**Comment by d4m** [comment]
Ok, I changed it.

**Comment by szczukot** [comment]
Looks good (workaround for RGB).

**Comment by gizmocuz** [comment]
Thanks!!!

**Pull request merged**